app/views/admin/layouts/_area.html.erb

```diff
@@ -0,0 +1,12 @@
+<% x = area.coordinate.nil? ? 0 : area.coordinate.x %>
```

Comment:
You could make use of the safe access operator
```ruby
x = area.coordinate&.x || 0
```

Reply:
This is changing in admin-ui-2

Reply:
Made a respective change in admin-ui-2